Doc/c-api/hash.rst

```diff
@@ -46,3 +46,12 @@ See also the :c:member:`PyTypeObject.tp_hash` member.
    Get the hash function definition.
 
    .. versionadded:: 3.4
+
+
+.. c:function:: Py_hash_t PyHash_Pointer(const void *ptr)
+
+   Hash a pointer.
```

Comment:
The result can be larger than `_PyHASH_MODULUS` and smaller than `-_PyHASH_MODULUS`. Is it an issue? Should it be documented?

Reply:
No, that's not an issue. The modulus is only used for hashing `PyLong`, and numbers which hash the same way as `PyLong`.